Assignment2/Sixtrax-fromRDFtoJSON-LD.jsonld

```diff
@@ -0,0 +1,55 @@
+{
+	"@context":{
+			"class":"http://www.example.org/Class#",
+      		"element":"http://www.example.org/Elements#",
+			"pr":"http://www.example.org/Properties#",
+      		"msr":"http://www.example.org/Measurements#",
+      		"user":"http://www.example.org/Users#",
+			"xsd":"http://www.w3.org/2001/XMLSchema#",
+			"Class01":"class:Class01",
+      		"includes":"pr:includes",
+			"hasMeasurement":"pr:hasMeasurement",
+			"hasOwner":"pr:hasOwner",
+			"hasTemperature":"pr:hasTemperature",
+			"atTime":"pr:atTime",
+			"hasName":"pr:hasName",
+			"Sensor029":"element:Sensor029",
+			"Computer101":"element:Computer101",
+			"Measurement8401":"msr:Measurement8401",
+			"User10A":"user:User10A"
+	},
+		"@id":"http://www.example.org/Class01",
+		"@type":"class:Class01",
+		"includes":[{
+			"@id":"element:Sensor029/",
+			"@type":"element",
+          	"hasMeasurement":{		
+				"Measurement8401":{
+					"@id":"http://www.example.org/Class01/Sensor029/Measurement8401",
+					"@type":"msr:Measurement8401",
+					"hasTemperature":{
+						"@value":"29",
+						"@type":"xsd:integer"
+					},
+					"atTime":{
+						"@value":"2010-06-12T12:00:12",
+						"@type":"xsd:dateTime"
+					}
+				}
+            }
+		},
+		{
+			"@id":"element:Computer101/",
+			"@type":"element",

```

Comment:
Please, change this type of computer resource since you have used it on the sensor type.